indico/modules/events/timetable/controllers/manage.py

```diff
@@ -167,3 +170,17 @@ def _process_PATCH(self):
             if colors not in get_colors():
                 raise BadRequest
             self.break_.colors = colors
+
+
+class RHCloneContribution(RHManageTimetableBase):
+    def _process_args(self):
+        RHManageTimetableBase._process_args(self)
+        self.contribution = (Contribution.query.with_parent(self.event)
+                             .filter_by(id=request.args['contrib_id'])
+                             .first())
+
+    def _process(self):
+        clone_contribution(self.event, self.contribution, True)
+        db.session.commit()
```

Comment:
why are you committing here? `db.session.flush()` is all you need if you just need to populate the id and default args